src/pdc-api.ts

```diff
@@ -48,17 +48,6 @@ const usePdcApi = <T>(
 			.then((res) => res.json())
 			.then(setResponse)
 			.catch((e) => logError(e, path, params));
-
-		/* eslint-disable-next-line react-hooks/exhaustive-deps --
```

Comment:
These rules are no longer included in our linting configuration with the react-scripts ejected, so the linter asked me to remove them. I figured if we're not disabling the rule, we no longer need the explanatory comment!